packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -62,6 +62,14 @@ export type ProductBlockInstance = {
     subscription: Pick<Subscription, 'subscriptionId' | 'description'>;
 };
 
+export type ProductBlockInstanceForDropdown = {
+    id: number;
+    subscriptionInstanceId: string;
+    parent: Nullable<number>;
+    productBlockInstanceValues: FieldValue[];
+    subscription: Pick<Subscription, 'subscriptionId' | 'description'>;
+};
+
```

Comment:
Is this the same type as ProductBlockInstance, but without the inUseByRelations prop? If yes, it might be better to use the Omit utility type

Reply:
Done!